docs/kyma/docs/05-01-overview.md

```diff
@@ -0,0 +1,63 @@
+## Installation configuration
+
+Before you start the Kyma installation process, you can customize the default settings in the installation and configuration files.
+
+### Components
+
+One of the installation artifacts is the Kyma Installer image that contains all component charts located in the `kyma/resources` folder. Both cluster and local installation files contain a full list of these component names and their Namespaces.
+Components that are not an integral part of the default Kyma Lite package, are preceded with hashtags (#) which means that the Installer skips them during the installation process.   
+You can customize the component installation files by:
+- Removing hashtags in front of the component entries to enable a component installation.
+- Adding hashtags to disable a component installation.
+- Adding new components to the list along with their chart definition in the `kyma/resources` folder. In that case you must create your own [Kyma Installer image](#configuration-use-your-own-kyma-installer-image) as you are adding new chart configuration.
+
+For more details on custom component installation, see [this](#configuration-custom-component-installation) document.
+
+### Overrides
+
+The configuration files provide production-ready settings for the Installer. These include ConfigMaps and Secrets with the overrides for the values hardcoded in the `values.yaml` files of component charts.
+
+You can modify local settings, such as memory limits for a given resource, by changing values in the existing ConfigMaps and Secrets or adding new ones. Every ConfigMap and Secret that the Installer reads must contain an `installer: overrides` label and a `component: {component-name}` label if it refers to a specific component. Add the `kyma-project.io/installation: ""` label to all non-Kubernetes resources to delete or search for them easier.
+
+[Read more](#configuration-helm-overrides-for-kyma-installation) about the types of overrides and the rules for creating them.
+
+## Runtime configuration
+
+Upon installing Kyma, you can still modify the installation artifacts. However, for these changes to take effect, you must trigger the update process:  
+
+```
+kubectl label installation/kyma-installation action=install
+```
+
+>**NOTE:** You cannot uninstall a Kyma component that is already installed. If you remove it from any of the installation files or add hashtags in front of its `name` and `namespace` entries, you only disable its further updates.
+
+Apart from modifying the installation artifacts, you can additionally configure the installed components and their behavior in the runtime using ConfigMaps and Secrets.
+
+For an example of such a runtime configuration, see [Helm Broker configuration](/components/helm-broker/#configuration-configuration) in which you add links the bundle repositories to a ConfigMap and label it with the `helm-broker-repo=true` label for the Helm Broker to expose additional Service Classes in the Service Catalog.
```

Comment:
` in which you add links the bundle repositories to a ConfigMap and label it with the `helm-broker-repo=true` label`

should it be ` in which you add links to the bundle repositories to a ConfigMap`?  (missing preposition)